app/src/main/res/layout/activity_save_book.xml

```diff
@@ -67,14 +67,41 @@
             app:layout_constraintStart_toStartOf="@+id/guidelineLeft"
             app:layout_constraintTop_toBottomOf="@+id/titleEditText"/>
 
+        <EditText
+            android:id="@+id/yearEditText"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/spacing_normal"
+            android:background="@android:color/transparent"
+            android:hint="Year"
```

Comment:
Another benefit of using _strings.xml_ instead of hardcoding them in layout file is that when you have to change them you do it only in one place. Now when you decide to replace "Year" with "Release year" in how many places do you have to make this change?

Reply:
This is true, thanks for the feedback. I will have the changes finished as soon as possible.